packages/icons/src/filestore.svg

```diff
@@ -0,0 +1,10 @@
+<svg viewBox="0 0 65 64" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M9.38281 45.5545L9.38281 55.0027C9.38281 55.2788 9.60667 55.5027 9.88281 55.5027H32.3828H54.8828C55.159 55.5027 55.3828 55.2788 55.3828 55.0027V45.5545" stroke="#0278D5" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
if this is not an logo of a company, the remove the color and replace it with `currentColor`

Reply:
And do this for all the colors in this files